python/cudf/cudf/tests/test_csv.py

```diff
@@ -1117,14 +1117,12 @@ def test_csv_reader_index_col():
     # using a column index
     cu_df = read_csv(StringIO(buffer), header=None, index_col=0)
     pd_df = pd.read_csv(StringIO(buffer), header=None, index_col=0)
-    for cu_idx, pd_idx in zip(cu_df.index, pd_df.index):
-        assert str(cu_idx) == str(pd_idx)
+    assert_eq(cu_df.index.tolist(), pd_df.index.tolist())
```

Comment:
Why do we need to call `tolist()` on these indexes to compare equality?

Reply:
Looks like we had a bug here, fixed it:

```python
(Pdb) pd_df.index
Int64Index([0, 3, 6], dtype='int64', name=0)
(Pdb) cu_df.index
Int64Index([0, 3, 6], dtype='int64', name='0')
```